boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -1251,6 +1251,29 @@ LoadAndBootWindowsCommon(
     /* "Stop all motors", change videomode */
     MachPrepareForReactOS();
 
+    /* Show the "debug mode" notice if needed */
+    BOOLEAN DebugMode = !NtLdrGetOption(BootOptions, "NODEBUG") && /* NODEBUG is absent, priority over DEBUG option */
+                        !NtLdrGetOption(BootOptions, "CRASHDEBUG") && /* CRASHDEBUG is abset, no wait during the boot */
+                        !!NtLdrGetOption(BootOptions, "DEBUG"); /* DEBUG is present */
+    if (DebugMode)
+    {
+        /* It is booting in debug mode */
+        TuiPrintf("Booting in debug mode\n");
+
+        /* Check whether there is a DEBUGPORT option */
+        PCSTR DebugPort;
+        ULONG DebugPortLength;
+        DebugPort = NtLdrGetOptionEx(BootOptions, "DEBUGPORT=", &DebugPortLength);
+        if (DebugPort && (DebugPortLength > 10))
```

Comment:
```suggestion
        if (DebugPort != NULL && DebugPortLength > 10)
```

Reply:
This follows the code style from other parts of free loader codebase. Actually I copy-pasted this part from `LoadWindowsCore` in this same file.
https://github.com/reactos/reactos/blob/d06d9f3e8f63e5a4b610e9605b40da20990e23a2/boot/freeldr/freeldr/ntldr/winldr.c#L809